### DISCUSSION TITLE
Выгрузка данных по API из Wildberies и Ozon

### DISCUSSION THREAD

**Discussion by Natalia-Valter** [totumonline/totum-ru-issues-and-discussions#190] [Как это сделать?] [answered]
Добрый день! Подскажите есть ли такая возможность в Totum как выгрузка заказов по API из Личного кабинета ВБ и Озона?
Если да, то есть ли инструкция как это можно реализовать?

**Comment by ttm-alexeypolunin** [comment, marked as answer]
Можно, так как у Тотума свободно-программируемое API, но готового решения «из коробки» как для amoCRM https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/169 еще не делали.

Следующее — это Битрикс24, потом сделаем Ozon

**Reply by ttm-alexeypolunin** [reply]
Ставьте звездочку основному репо https://github.com/totumonline/totum-mit кому это нужно...

**Reply by Natalia-Valter** [reply]
> Можно, так как у Тотума свободно-программируемое API, но готового решения «из коробки» как для amoCRM #169 еще не делали.
> 
> Следующее — это Битрикс24, потом сделаем Ozon

А если мы отчеты будем скидывать например на гугл диск, а оттуда загружать данные в тотум

**Reply by ttm-alexeypolunin** [reply]
Лучше тогда загружать сразу в Тотум: https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/191

Но тут вопрос не в загрузке файла, а в том, как его разобрать. Если там `.csv` или любой другой `.txt` с понятными разделителями — то ок. Если там `.xlsx`, то Тотум их сможет обрабатывать через пару-тройку месяцев.

Но вообще, у Озона прекрасная документация на их API, другой вопрос, что там огромный объем и **сделать синхронизацию всего** — это нужна команда, сравнимая с командой озона.

Если вы напишете конкретно, что вы хотите выгрузить, то я не думаю, что это как-то экстра-сложно.

**Reply by ShilGen** [reply]
> Добрый день! Подскажите есть ли такая возможность в Totum как выгрузка заказов по API из Личного кабинета ВБ и Озона? Если да, то есть ли инструкция как это можно реализовать?

Наталья, мой ник в тг @shilgen - я тоже продавец на озон, хочу на вб, могу тратить час в день на прикрутку их апи к тотуму. напишите мне - обсудим.

**Comment by Natalia-Valter** [comment]
Здравствуйте подскажите вы уже себе настраивали выгрузку данных по Апи из Озона (ВБ)? это возможно делать автоматически каждый час? или только вручную? -- С уважениемНаталья   14.08.2022, 03:09, "Shilov Gennady" ***@***.***>: Добрый день! Подскажите есть ли такая возможность в Totum как выгрузка заказов по API из Личного кабинета ВБ и Озона? Если да, то есть ли инструкция как это можно реализовать?Наталья, мой ник в тг @ShilGen - я тоже продавец на озон, хочу на вб, могу тратить час в день на прикрутку их апи к тотуму. напишите мне - обсудим.—Reply to this email directly, view it on GitHub, or unsubscribe.You are receiving this because you authored the thread.Message ID: ***@***.***>

**Reply by ttm-alexeypolunin** [reply]
Минимальный период Крона (задания по расписанию) — 1 минута.
